### PR TITLE
add content to github_and_markdown

### PR DESCRIPTION
**Pull request by augustinanders**
add heading

### CODE REVIEW

**Review by manukao — CHANGES_REQUESTED** [review]
Looks a bit empty here my friend :D

File: shell-and-git/sessions/github_and_markdown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Github and Markdown
+
+## Markdown
```
Comment (rabe200) [inline shell-and-git/sessions/github_and_markdown.md:3]: no malicious code detected

**Review by toshydev — CHANGES_REQUESTED** [review]
Add more content and then LGTM :)

**Review by rabe200 — CHANGES_REQUESTED** [review]
add text

**Review by toshydev — APPROVED** [review]
LGTM! 👾

**Pull request merged**